web/packages/teleport/src/UnifiedResources/Resources.tsx

```diff
@@ -182,8 +181,21 @@ export function Resources() {
   );
 }
 
-function resourceKey(resource: UnifiedResource) {
-  return `${resource.kind}/${resourceName(resource)}`;
+export function resourceKey(resource: UnifiedResource) {
+  if (resource.kind === 'app') {
+    return `${resource.name}/app`;
+  }
```

Comment:
I think we only need an exception for `resource.kind === 'node'` (we have `resource.name` and `resource.kind` defined for apps).

Reply:
Rebased after i updated just because it was a small change